### PR TITLE
Fix MangaDex.

### LINKED ISSUE #13431: Mangadex HTTP error 503
### Source information and language

Mangadex

### Steps to reproduce

Just try to open the source.

Here's a video proof
https://youtube.com/shorts/fDr9qHmnCnY

### Expected behavior

Expected to show mangas from mangadex.

### Actual behavior

Error 503 instead and didn't show mangas.

### Tachiyomi version

0.13.6

### Android version

Android 13

### Other details

I thought Mangadex was having a maintenance, but when i checked using browser, Mangadex works.
Using google dns, on my router, in tachiyomi app settings, and even in android's connection settings.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.13.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by ReimuHakurei**
MangaDex is currently blocking Tachiyomi. This changes the User-Agent to that of Chrome for Android to evade the block. Fixes #13431

Checklist:

- [X] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [X] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [X] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [X] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [X] Have not changed source names
- [X] Have explicitly kept the `id` if a source's name or language were changed
- [X] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

**Comment by ghostbear** [comment]
> MangaDex is currently blocking Tachiyomi.

We are fully aware of that and respect their decision to block Tachiyomi as it's a temporarily block until they have stabilized their infrastructure 

From the MangaDex Discord:

> TL;DR: The issue is (still) not properly fixed on our end, but the site should be at least accessible and reasonably working for everyone for now.
The main way we have this is by a temporary measure, which was blocking Tachiyomi from accessing MangaDex.
To be clear, this is temporary and purely to buy us a little bit more time without needing to keep parts of the world blocked (notably NA east).
We don't want to block third-parties (and in this case friends, for many within the staff), but we don't have much choice as things stand.
We have obviously let them know already, and we will notify you as soon as we have unblocked it.
